docs/sources/k6/next/javascript-api/k6-browser/frame/locator.md

```diff
@@ -1,18 +1,65 @@
 ---
-title: 'locator(selector)'
-description: 'Browser module: frame.locator(selector) method'
+title: 'locator(selector[, options])'
+description: 'Browser module: frame.locator(selector[, options]) method'
 ---
 
-# locator(selector)
+# locator(selector[, options])
 
 The method returns an element [Locator](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-browser/locator/). Locators resolve to the element when the action takes place, which means locators can span over navigations where the underlying dom changes.
 
-| Parameter | Type   | Default | Description                                   |
-| --------- | ------ | ------- | --------------------------------------------- |
-| selector  | string | `''`    | A selector to use when resolving DOM element. |
+<TableWithNestedRows>
```

Comment:
I don't think we have this element in the Grafana docs. Do you know where this came from? 🤔

Reply:
Hmm, I don't know. Other markdowns in his folder have it.